QSB/WorldRegistry.cs

```diff
@@ -0,0 +1,39 @@
+﻿using QSB.Events;
+using QSB.Utility;
+using QSB.WorldSync;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace QSB
+{
+    public static class WorldRegistry
+    {
+        public static List<GeyserController> GeyserControllers = new List<GeyserController>();
+
+        public static void GenerateComponentList()
+        {
+            GeyserControllers = Resources.FindObjectsOfTypeAll<GeyserController>().ToList();
+
+            foreach (var component in GeyserControllers)
```

Comment:
Wanna pull some of this stuff out and put in GeyserManager or similar, like in elevator sync? Would make WorldRegistry cleaner